moveit_ros/benchmarks/scripts/moveit_benchmark_statistics.py

```diff
@@ -130,6 +130,18 @@ def readBenchmarkLog(dbname, filenames):
         (runid INTEGER, time REAL, PRIMARY KEY (runid, time),
         FOREIGN KEY (runid) REFERENCES runs(id) ON DELETE CASCADE)""")
 
+    # add placeholder entry for all_experiments
+    allExperimentsName = "all_experiments"
+    allExperimentsValues = {
+        "totaltime": 0.0, "timelimit": 0.0, "memorylimit": 0.0, "runcount": 0, "version": "0.0.0",
+        "hostname": "",   "cpuinfo": "",    "date": 0,          "seed": 0,     "setup": ""
```

Comment:
I like this idea, but the other fields of `allExperimentsValues` should be init too: `timelimit`/`memorylimit` can just be the max over all the runs, version, hostname, cpuinfo, and date should at least be something, maybe just the first file?